composite/action.yml

```diff
@@ -175,8 +175,9 @@ runs:
         esac
       shell: bash
 
-    - name: Cache PIP Packages
-      uses: actions/cache@v3
+    - name: Restore PIP packages cache
+      id: cache
+      uses: actions/cache/restore@v3
```

Comment:
can I use this branch directly?

Reply:
Let's see https://github.com/TWiStErRob/net.twisterrob.libraries/pull/38/commits/b66a8c6c695b88f4e359c275b9298d36bdb7a4cd

Reply:
```
uses: EnricoMi/publish-unit-test-result-action/composite@branch-composite-cache-pip-packages-2
```